<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeWindow</class>
 <widget class="QWidget" name="WelcomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_welcome">
   <item>
    <widget class="QLabel" name="welcomeLabel">
     <property name="text">
      <string>Welcome!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="userFirstNameLabel">
     <property name="text">
      <string>First Name: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="userLastNameLabel">
     <property name="text">
      <string>Last Name: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="userIDLabel">
     <property name="text">
      <string>User ID: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="incomeButton">
     <property name="text">
      <string>Income</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="expenseButton">
     <property name="text">
      <string>Expense</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="totalButton">
     <property name="text">
      <string>Total</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
